pkg/apis/seedmanagement/v1alpha1/helper/helper.go

```diff
@@ -59,3 +59,18 @@ func ExtractSeedTemplateAndGardenletConfig(managedSeed *seedmanagementv1alpha1.M
 
 	return seedTemplate, gardenletConfig, nil
 }
+
+// ExtractGardenletConfig extracts the gardenlet config from seed
+func ExtractGardenletConfig(managedSeed *seedmanagementv1alpha1.ManagedSeed) (*configv1alpha1.GardenletConfiguration, error) {
```

Comment:
Add a unit test?

Reply:
Probably you should make the `ExtractSeedTemplateAndGardenletConfig` call this new `ExtractGardenletConfig` function now.

Reply:
Done